application/backend/app/services/training_configuration_service.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+from uuid import UUID
+
+from sqlalchemy.orm import Session
+
+from app.configuration_tools.training_configuration import TrainingConfiguration
+from app.repositories import ModelRevisionRepository, ProjectRepository
+from app.repositories.training_configuration_repo import TrainingConfigurationRepository
+from app.services import ResourceNotFoundError, ResourceType
```

Comment:
```suggestion
from .base import ResourceNotFoundError, ResourceType
```
Prefer relative imports when at the same level, to prevent circular imports between services.